gtsfm/loader/loader_base.py

```diff
@@ -143,3 +143,12 @@ def get_valid_pairs(self) -> List[Tuple[int, int]]:
                     indices.append((idx1, idx2))
 
         return indices
+
+    def get_image_dictionary(self) -> Dict[int, Image]:
```

Comment:
looks good @codyly . Can you add a unit test on the Lund door set 1 data in tests/data?

Reply:
actually, we already have this functionality in 
```
            image_graph=loader.create_computation_graph_for_images(),
```
right?

Reply:
so I dont think we need this PR, after all, if we are working within Dask

Reply:
Ah...I think so. I am not very familiar with Dask so I didn't realize it.

Reply:
where are you hoping to use this function @codyly ?

Reply:
in the densify implementation at #195

Reply:
After we get `sfm_result` in `run_scene_optimizer`, `sfm_result` together with `images_dict` will be passed to `MVSPatchmatchnet.densify` (implemented in #195)

Reply:
@johnwlambert As the `MVSBase` defines the densify API as
```def densify(self, images: Dict[int, Image], sfm_result: GtsfmData) -> np.ndarray:```
Is there a simple way to transfter image list from `loader.create_computation_graph_for_images()` to image dictionary, such as
```dict(zip(list(range(len(loader))), loader.create_computation_graph_for_images()))```